cpp/include/cudf/column/column_factories.hpp

```diff
@@ -330,7 +331,7 @@ std::unique_ptr<column> make_fixed_width_column(
 }
 
 /**
- * @brief Construct STRING type column given a device vector of pointer/size pairs.
+ * @brief Construct STRING type column given a device span of pointer/size pairs.
```

Comment:
Add a blank line after the `@brief` tag.
```suggestion
 * @brief Construct STRING type column given a device span of pointer/size pairs.
 
```
https://github.com/rapidsai/cudf/blob/branch-0.19/cpp/docs/DOCUMENTATION.md#brief

Reply:
I think the blank line needs to start with a `*`.

Reply:
Oops, right.